internal/pkg/controllers/profile/profile.go

```diff
@@ -110,23 +110,19 @@ func (r *Reconciler) Reconcile(req reconcile.Request) (reconcile.Result, error)
 
 	for profileName, profileContent := range configMap.Data {
 		if err := validateProfile(profileContent); err != nil {
-			reason := "cannot validate profile " + profileName
-			logger.Error(err, reason)
-			r.record.Event(configMap, event.Warning(event.Reason(reason), err))
+			r.pushEventForConfigMap(logger, configMap, "cannot validate profile", err)
```

Comment:
Hmm currently we are only emitting events for errors, but I don't think this would be the case in the future. I am inclined to keep the code a bit more verbose since this change is only removing one line in each instance. What do you think?

Reply:
Hm yeah I agree. Let's close this for now :+1: